scripts/build/rollup-demos.js

```diff
@@ -50,8 +50,6 @@ const rollupOptions = {
       module: true,
       jsnext: true,
       main: true,
-      browser: true,
-      modulesOnly: true,
```

Comment:
Why were these true originally? I'm having trouble figuring that out.

Reply:
Copy/paste error 🙈